### ISSUE TITLE
Accessibility - UI5 WC FlexibleColumnLayout not accessible for 200% screen zoom

### ISSUE THREAD

**Issue by Teju-SFSF** [SAP/ui5-webcomponents#3321]
### **Bug Description**
When the screen size is zoomed to 200%, then all the columns in FlexibleColumnLayout are not visible and not accessible. Only one column is displayed.

### **Expected Behavior**
When the screen size is zoomed to 200%, then all the columns in FlexibleColumnLayout should be visible and accessible. 

### **Steps to Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents-react/?path=/docs/layouts-floorplans-flexiblecolumnlayout--default-story
2. Increase zoom in the browser screen to 200%.
3. All columns are not visible or accessible.

### **Context**
- OS/Platform: Windows, Mac
- Browser: IE, Chrome
- Affected component: UI5 WC FlexibleColumnLayout

### **Log Output / Stack Trace / Screenshots**
![image](https://user-images.githubusercontent.com/84871522/119745762-9f527680-be43-11eb-967a-ee3a993b6561.png)
![image](https://user-images.githubusercontent.com/84871522/119745794-b5f8cd80-be43-11eb-9174-acafb6d49d42.png)

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

**Comment by tsanislavgatev** [comment]
Hello colleagues,

Can you please take a look at this issue?

Best Regards,
Tsanislav

**Comment by nnaydenow** [comment]
Hi @Teju-SFSF,

As you can see in the UX specification of the ui5-flexible-column-layout (https://experience.sap.com/fiori-design-web/flexible-column-layout/#size-m-tablet) there is different column layout due to limited screen width.

Regards,
Nayden

**Comment by Teju-SFSF** [comment]
Thank you Nayden. So is there any plan for supporting 200% zoom for the component in the near future?

Thanks,
Teju

**Comment by nnaydenow** [comment]
Hi @Teju-SFSF,

In S breakpoint only 1 column should be visible by UX specification this is the reason why arrows are not shown. The application should be handled by application side as you can see in our examples here: https://sap.github.io/ui5-webcomponents/playground/components/FlexibleColumnLayout/ (by clicking table rows or something else).

@ilhan007 can you check if I'm missing something?

Regards,
Nayden

**Comment by nnaydenow** [comment]
Hi @Teju-SFSF,

When you change the zoom ratio it also change the content size. When the content size is change the breakpoint is changed too because the breakpoint is based to the component width, not on the screen size. 

There is property which allows you to do customization of the layout on your own. You can check #1972 for more detailed information how to do that.

Regards,
Nayden
